change/@microsoft-teams-js-9c1b77aa-a0e6-4099-9eae-9fff30c8f94e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Dialog.open function takes an extra optionl parameter named messageForChildHandler and retuns a handler back instead of a ChildAppWindow. ",
```

Comment:
I will fix the `return` spelling. just noticed now

Reply:
You can do that inside of Github itself if you'd like